circleguard/enums.py

```diff
@@ -11,34 +11,34 @@ class Error(Enum):
     UNKOWN            = ["Unkown error.", UnkownAPIException, "Unkown error when requesting replay. Please lodge an issue with the devs immediately"]
 
 class Mod(Enum):
-    NoMod          = 0
-    NoFail         = 1
-    Easy           = 2
-    NoVideo        = 4
-    Hidden         = 8
-    HardRock       = 16
-    SuddenDeath    = 32
-    DoubleTime     = 64
-    Relax          = 128
-    HalfTime       = 256
-    Nightcore      = 512
-    Flashlight     = 1024
-    Autoplay       = 2048
-    SpunOut        = 4096
-    Autopilot      = 8192
-    Perfect        = 16384
-    Key4           = 32768
-    Key5           = 65536
-    Key6           = 131072
-    Key7           = 262144
-    Key8           = 524288
-    keyMod         = 1015808
-    FadeIn         = 1048576
-    Random         = 2097152
-    LastMod        = 4194304
-    TargetPractice = 8388608
-    Key9           = 16777216
-    Coop           = 33554432
-    Key1           = 67108864
-    Key3           = 134217728
-    Key2           = 268435456
+    NoMod               = 0
```

Comment:
inconsistent spacing (line up all the 2 letter names like the numbers are)

also all mods should be getting acronyms (might as well), use the ones from here https://github.com/christopher-dG/osu-bot/blob/master/osubot/consts.py#L103 for consistency if he has them, or make them up if he doesn't (NV for no video, K1 - K9)

Reply:
Yea, will reformat it & add the missing ones (for now I just did the ones on the osu! wiki, didn't want to make anything stupid)
Gonna make the changes tommorow.

Reply:
does this resolve it tybug

Reply:
It does yeah, sorry for the late review, had a lot going on yesterday.